migrations/20201210150000-fix-host-payment-methods-and-transactions.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    const [, updateTs] = await queryInterface.sequelize.query(`
+      UPDATE "Transactions"
+      SET "amountInHostCurrency" = "amount", "hostCurrencyFxRate" = 1
+      FROM "PaymentMethods" pm, "Collectives" c
+      WHERE
+        "Transactions"."currency" = "Transactions"."hostCurrency"
+        AND "Transactions"."hostCurrencyFxRate" != 1
+        AND pm.id = "Transactions"."PaymentMethodId"
+        AND c.id = pm."CollectiveId"
+        AND pm."type" = 'host'
+        AND pm."currency" != c."currency"
```

Comment:
I think the condition should be on "Transactions"."currency" to make sure we don't have side effects in cases we're not interested.